test/wallet-rpc-test.js

```diff
@@ -41,14 +43,17 @@ const node = new FullNode({
 const {wdb} = node.require('walletdb');
 
 const nclient = new NodeClient({
+  host: '127.0.0.1',
   port: ports.node,
   apiKey: 'bar',
   timeout: 15000
 });
 
 const wclient = new WalletClient({
+  host: '127.0.0.1',
   port: ports.wallet,
-  apiKey: 'bar'
+  username: 'foo',
+  password: 'bar'
```

Comment:
This came from cross-testing with bitcoind. `apiKey === password`. Should I switch this back ?

Reply:
yeah I think so. We recommend 64-char random hex strings as api keys, thats not really a password.